app/models/beta/search/intercept_message.rb

```diff
@@ -10,14 +41,27 @@ class InterceptMessage
       def self.build(search_query)
         query = search_query.downcase
 
-        return unless query.eql?(I18n.t("#{query}.title"))
+        return if I18n.t("#{query}.message", default: nil).blank?
+
+        intercept_message = new
 
-        result = new
+        message = I18n.t("#{query}.message")
+        message = message.ends_with?('.') ? message : message.concat('.')
+
+        intercept_message.term = query
+        intercept_message.message = message
```

Comment:
it'd be nice to be able to do
`intercept_message = new(query, message)`

maybe we need the `def initialize(query, message)`